doc/xml/user-guide.xml

```diff
@@ -866,7 +866,7 @@
     <section id="installation">
         <title>Installation</title>
 
-        <p>A new host named <host>pg1</host> is created to contain the demo cluster and run <backrest/> examples.</p>
+        <p>A new host named <host>{[host-pg1]}</host> is created to contain the demo cluster and run <backrest/> examples.</p>
```

Comment:
This is an improvement, but I'd like to see us not use `pg1` at all, to make it clear it is absolutely unrelated to things like `pg1-port`. Maybe `postgres-node1` or similar?

Reply:
Hm, afaik, we're not using `pg1` -> `<variable key="host-pg1">pg-primary</variable>`
That's why `pg-primary` is displayed in the user-guide...

Reply:
Right, `pg-*` is just the name of the internal variable, not what is displayed to the user (`pg-primary`, `pg-standby`), except for that one typo. Renaming the variables would cause a lot of unneeded churn, in my opinion.

Reply:
@turnstep I think whether or not we rename these variables should be discussed in #1476. This PR is pretty much just a straight bug fix and I'd like to get it into this release.